frontend/src/features/properties/map/MapView.tsx

```diff
@@ -42,8 +42,10 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
     history.push(`/mapview/sidebar/property/${property.id}?pid=${property.pid}`);
   };
 
-  const onPropertyViewClicked = (pid?: string | null) => {
-    if (pid !== undefined && pid !== null) {
+  const onPropertyViewClicked = (pid?: string | null, id?: number) => {
+    if (id !== undefined) {
+      history.push(`/mapview/sidebar/property/${id}?pid=${pid}`);
```

Comment:
pid is not guaranteed to be non null here

Reply:
having a null pid works here, I've tested it. it just results in the ltsa screen not being shown.

Reply:
the url ends up with pid=